modules/con-sdpolicy-account-management.adoc

```diff
@@ -66,29 +55,13 @@ Compute-optimized
 - C5.2xlarge (8 vCPU, 16 GiB)
 - C5.4xlarge (16 vCPU, 32 GiB)
 
-== Google Cloud compute types
-
-{product-title} offers the following worker node types and sizes on GCP:
-
-General purpose
-
-- custom-4-16384 (4 vCPU, 16 GiB)
-- custom-8-32768 (8 vCPU, 32 GiB)
-- custom-16-65536 (16 vCPU, 64 GiB)
-
-Memory-optimized
-
-- custom-4-32768-ext (4 vCPU, 32 GiB)
-- custom-8-65536-ext (8 vCPU, 64 GiB)
-- custom-16-131072-ext (16 vCPU, 128 GiB)
-
-Compute-optimized
-
-- custom-8-16384 (8 vCPU, 16 GiB)
-- custom-16-32768 (16 vCPU, 32 GiB)
-
 == Regions and availability zones
-The following AWS regions are supported by Red Hat OpenShift 4 and are supported for {product-title}. Note: China and GovCloud (US) regions are not supported, regardless of their support on OpenShift 4.
+The following AWS regions are supported by Red Hat OpenShift 4 and are supported for {product-title}.
+
+[NOTE]
+====
+China and GovCloud (US) regions are not supported, regardless of their support on OpenShift 4.
```

Comment:
If this were for OCP, I'd tell you to change it. We don't usually talk about generic OpenShift there, but this seems reasonable.